src/ui/components/forms/VPNInputMessages.qml

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.6
+import QtQuick.Layouts 1.14
+import Qt.labs.qmlmodels 1.0
+
+import themes 0.1
+
+ColumnLayout {
+    id: messagesContainer
+    property var messages
+
+    anchors.left: parent.left
+    anchors.right: parent.right
+    anchors.topMargin: Theme.listSpacing
+    spacing: Theme.listSpacing / 4
+
+    DelegateChooser {
+        id: messagesChooser
+        role: "type"
+
+        DelegateChoice {
+            roleValue: "info"
```

Comment:
Does this get used anywhere?

Reply:
Currently this is not used anywhere. If I’m not mistaken I saw use for it in a WIP version of the [in-app sign in/auth design](https://www.figma.com/file/gSRPHqjzqjlMA9rXDc5SqH/VPN_InApp_SignIn%2FAuth?node-id=1%3A708) — I’ll remove it for now.

Reply:
This message type is currently being used. If I’m not mistaken I have seen use for the `info` type in an earlier version of the [in-app auth spec](https://www.figma.com/file/gSRPHqjzqjlMA9rXDc5SqH/VPN_InApp_SignIn%2FAuth?node-id=1%3A708). I’ll remove this for now.